client/src/ui/molecules/language-toggle/index.scss

```diff
@@ -4,11 +4,21 @@
 
 ul.language-toggle {
   display: flex;
-  justify-content: flex-end;
-  padding: $base-spacing;
+  text-align: right;
 
-  @media #{$mq-small-desktop-and-up} {
-    justify-content: inherit;
+  @media #{$mq-tablet-and-up} {
+    justify-content: flex-end;
+    position: relative;
+    text-align: inherit;
+    top: -2px;
```

Comment:
Why a `px` value?

Reply:
For very specific positioning like this, I find it best to use an absolute value such as pixels as opposed to a relative unit like say `rem` or percentage.

Reply:
To be honest, I am not very fond of this solution. Perhaps experimenting with the `line-height` might be better. It works though so, I reckon it is ok for now.